File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_19.go
package main
import ("fmt"
        
)

func main() {
    var x int
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&x)
    c := x % 10
    b := x / 10 % 10
    a := x / 100
    if a == b {
        fmt.Print ("TRUE")
    } else if a == c {
        fmt.Print ("TRUE")
    } else if b == c {
        fmt.Print ("TRUE")
    } else if (a == b) && (b == c) && (a == c) {
        fmt.Print ("TRUE")
    } else {
        fmt.Print ("FALSE")
    }
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_19.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
x = gets.to_i
c = x % 10
b = x / 10 % 10
a = x / 100
if (a == b)
  print ('TRUE')
elsif (a == c)
  print ('TRUE')
elsif (b == c)
  print ('TRUE')
elsif (a == b) & (b == c) & (a == c)
  print ('TRUE')
else
  print ('FALSE')
end